app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -207,6 +208,19 @@ def initialize_edit_form
       @groups = current_user.groups
     end
 
+    # Retrieves the admin set options for the file set
```

Comment:
Should make it clear in the comment that it is only retrieving the admin set that the fileset belongs to